pkg/ebpf/verifier/calculator/main.go

```diff
@@ -0,0 +1,57 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package main exercises BuildVerifierStats and outputs the result as a JSON to stdout
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/ebpf/verifier"
+)
+
+func main() {
+	var objectFiles []string
+	var err error
+
+	if len(os.Args[1:]) < 1 {
+		panic("please use './main <object-files-dir>'")
+	}
+	directory := os.Args[1]
+
+	if err := filepath.WalkDir(directory, func(path string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+		if d.IsDir() {
+			return nil
+		}
+
+		if strings.Contains(path, "-debug") || !strings.HasSuffix(path, ".o") {
```

Comment:
Could we have an option to allow stats in debug builds be reported too? I think it could be useful in some situations.

Reply:
Good point. I will add support for this.